handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -256,6 +256,13 @@ public static boolean isAlpnSupported() {
     }
 
     /**
+     * Returns {@code true} if the used version of OpenSSL supports OCSP stapling.
+     */
+    public static boolean isOcspSupported() {
+      return version() >= 0x10002000L;
```

Comment:
I believe the `#define` is coming from here. Not sure what the best course of action is to unset it.

https://github.com/google/boringssl/blob/master/include/openssl/opensslconf.h#L45